getusermedia.html

```diff
@@ -1794,12 +1794,12 @@ <h2>Constrainable Properties</h2>
               <td>{{double}}</td>
               <td>The exact aspect ratio (width in pixels divided by height in
               pixels, represented as a double rounded to the tenth decimal
-              place) or aspect ratio range.</td>
+              place).</td>
```

Comment:
@youennf wanted time to file an issue based on this language, so leaving it for now.
```suggestion
              place) or aspect ratio range.</td>
```